aptos-move/framework/aptos-framework/sources/storage_gas.move

```diff
@@ -209,32 +233,45 @@ module aptos_framework::storage_gas {
         // Check the corner case that current_usage_bps drops before the first point.
         let (left, right) = if (num_points == 0) {
             (&Point {x: 0, y: 0}, &Point {x: BASIS_POINT_DENOMINATION, y: BASIS_POINT_DENOMINATION})
+        } else if (current_usage_bps < vector::borrow(points, 0).x) {
+            (&Point { x: 0, y: 0 }, vector::borrow(points, 0))
+        } else if (vector::borrow(points, num_points-1).x <= current_usage_bps) {
```

Comment:
nit: space before and after "-" -> num_points - 1

Reply:
Done.